src/interop/lib/get_mocha_tests.js

```diff
@@ -1,68 +1,20 @@
-var _ = require("lodash");
-var acorn = require("acorn");
-var walk = require("acorn/dist/walk");
-var path = require("path");
-var fs = require("fs");
+var path = require('path');
+var exec = require('sync-exec');
 
-var mochaSettings = require("./mocha_settings");
+var reporter = path.resolve(__dirname, 'test_capture.js');
+var mochaSettings = require('./mocha_settings');
+var mochaOpts = mochaSettings.mochaOpts;
+var mochaAtom = mochaSettings.mochaAtom;
```

Comment:
that's the next task - inspecting mochaAtom and then using this to segment the test suite. Aiming for 'browser' and 'test' to begin with, other possible future values could be 'file' or 'suite'. Feel free to suggest a new name for the configuration option :)